apps/package.json

```diff
@@ -154,6 +154,7 @@
     "which": "1.1.1"
   },
   "dependencies": {
+    "@kadira/storybook": "^1.41.0",
```

Comment:
You said this is upgrading, but it looks like an entirely new entry.  Was this not actually a dependency befoer?